src/app/public/modules/progress-indicator/progress-indicator-item/progress-indicator-item.component.ts

```diff
@@ -1,63 +1,79 @@
 import {
+  ChangeDetectionStrategy,
+  ChangeDetectorRef,
   Component,
   Input
 } from '@angular/core';
 
 import {
-  SkyProgressIndicatorDisplayMode
-} from '../types/progress-indicator-mode';
+  SkyProgressIndicatorItemStatus
+} from '../types';
 
 @Component({
   selector: 'sky-progress-indicator-item',
   templateUrl: './progress-indicator-item.component.html',
-  styleUrls: ['./progress-indicator-item.component.scss']
+  styleUrls: ['./progress-indicator-item.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush
 })
 export class SkyProgressIndicatorItemComponent {
-  public itemNumber: number;
-  public isActive = false;
-  public isComplete = false;
-  public isLastItem = false;
-  public displayMode = SkyProgressIndicatorDisplayMode.Vertical;
-  public isPassive = false;
-  public isNextToInactive = true;
 
   @Input()
   public title: string;
 
-  public get displayTitle(): string {
-    if (this.isPassive) {
-      return this.isActive || this.isComplete ? this.title : '';
-    } else {
-      return this.itemNumber + ' - ' + this.title;
-    }
+  public get status(): SkyProgressIndicatorItemStatus {
+    return this._status;
```

Comment:
I used to have a default value in there, but it was never `undefined` (since the status is set by the parent component with known status values). I'll add a default value just in case, and throw an `istanbul ignore else` so we can maintain 100% coverage.

Reply:
Should status have a default value?